### ISSUE TITLE
File page follow up improvements

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-desktop#3962]
- [ ] Move tip/support closer to total LBC value, move Follow near the channel.
- [ ] On download only content, we show a labeled download section, with a download button, and the download button from the main section
- [ ] spacing between cards too large? (feels like it on mobile at least)

**Comment by kamotswind** [comment]
Yes overall vertical spacing is too large. Also the fonts are much bigger then they need to be. The other action buttons could be moved up next to the Follow button which would be an immediate improvement.

The video window could also be wider. Pictures below showing some comparisons. These are all from a 16:9 1080P monitor.

https://cdn.discordapp.com/attachments/377895389992321064/700385442121646080/unknown.png

https://cdn.discordapp.com/attachments/377895389992321064/700388444098134016/unknown.png

https://cdn.discordapp.com/attachments/377895389992321064/700388487496728606/unknown.png
